cli/011-cli-multi-config.md

```diff
@@ -0,0 +1,74 @@
+- Start Date: 2021-08-22
+- RFC PR:
+- Issue:
+
+### Configuring multiple environments
+
+##### Problem
+
+Users may have multiple Temporal service environments (development, production, ..) with their own respective mTLS configs, namespaces, address etc. Current implementation of tctl config feature only supports configuring a single environment and doesn't help at quickly switching between multiple.
+
+##### Proposal
+
+To make it easy for users to switch between environments, allow configuring multiple instead of just one and provide command to switch between them.
+
+Syntax to activate environment configs:
+
+```shellsession
+$ tctl config use-environment <name>
+Active environment: <name>
+```
+
+If the environment doesn't yet exist in config, create it and set as active
```

Comment:
I'm missing a command to update an environment's configuration.

Reply:
i was thinking that `tctl config set <property> <value>` would update the currently active environment. Then if `--environment` [see link](https://github.com/temporalio/proposals/pull/39/files#diff-ac63135de4d849c82417254728c180bed798eeb90b570b5ae3248ccb9d0150bfR49) is passed, update the passed environment